howto/playAudio.md

```diff
@@ -1,16 +1,25 @@
-{% method %}
+# How To Play Audio On A Call {#top}
+
+## About {#about}
```

Comment:
Since we're doing more & more with BXML, what do you think about combining the tutorials that you've got going on on the ap-docs for bxml, and the ones here into a single 'tutorials, guide, building blocks' section that _then_ breaks down into REST API and BXML?

Reply:
That sounds good as long as its all consistent and the same format